ndcube/utils/towncrier.py

```diff
@@ -0,0 +1,81 @@
+"""
```

Comment:
I feel like we really can just import this from sunpy.

Reply:
#yourchoicebutwouldavoidaharddeponsunpyifyoujustwantobuildthedocs

Reply:
just skip writing out the changelog if you can't import sunpy. We have a hard dep on sunpy anyway at the moment, we can worry about refactoring it later.

Reply:
Sorted.

Reply:
Thanks!